online-unsafe-recovery.md

```diff
@@ -158,12 +164,36 @@ If an error occurs during the task, the last stage in the output shows `"Unsafe
 
 ### Step 3. Check the consistency of data and index (not required for RawKV)
 
-After the recovery is completed, the data and index might be inconsistent. Use the SQL commands [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md), `ADMIN RECOVER`, and `ADMIN CLEANUP` to check the consistency of the affected tables (you can get IDs from the output of `"Unsafe recovery finished"`) for data consistency and index consistency, and to recover the tables.
-
 > **Note:**
 >
 > Although the data can be read and written, it does not mean that there is no data loss.
 
+After the recovery is completed, the data and index might be inconsistent. Use the SQL command [`ADMIN CHECK`](/sql-statements/sql-statement-admin-check-table-index.md) to check the data consistency and index consistency of the affected tables
+
+```sql
+ADMIN CHECK TABLE table_name;
+```
+
+If there are inconsistent indexes, you can fix the index inconsistency by renaming the old index, creating a new index, and droping the old index.
```

Comment:
```suggestion
If there are inconsistent indexes, you can fix the index inconsistency by renaming the old index, creating a new index, or droping the old index.
```

Reply:
@v01dstar PTAL

Reply:
"and then followed by dropping the old index?"

It should be logical conjunction, and order matters.